### ISSUE TITLE
Offset

### ISSUE THREAD

**Issue by deladriere** [ridiculousfish/HexFiend#119]
I would we could to set an offset (in hex) instead of displaying byte from zero

**Comment by kainjow** [comment]
You can click the status bar text to rotate to hexadecimal mode.

**Comment by deladriere** [comment]
Yes ok to display in hex but what about  adding an offset so all the display  showed in the left column would start with that offset (not at 0000)

**Comment by kainjow** [comment]
Sorry, I misunderstood.

One thing I've wanted to do is better abstract what is a "file" in HF, so one would more easily work with data such as extended attributes of a file (which requires completely different reading and writing code) or other things down the road. If this were to happen then it could be nice to have a feature like you suggested where one could open a subset of a file and work on that.

**Comment by kainjow** [comment]
Closing, no plans to implement this.

**Comment by kainjow** [comment]
Of course, after I close this, with no other interest for 2 years, more requests come in :). I'm reopening, but still currently don't have plans for this. We can think about how it might work though.

**Comment by kainjow** [comment]
I think one thing that would be important is if you opened a subset of a file, it needs to be clear you're working on a subset, and you cannot extended that file. So the subset file backing must enforce this.
